### ISSUE TITLE
NH3+ N-terminus

### ISSUE THREAD

**Issue by sreisser** [mtien/PeptideBuilder#47]
Hi,
thanks for this nice piece of code!
I was wondering if it's possible to create the N-terminus with an NH3+ group?
Cheers
Sabine

**Comment by clauswilke** [comment]
I don't think so. You're welcome to create a pull request that adds this feature, though.
